src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx

```diff
@@ -61,11 +59,11 @@ const DictionaryDetailCard = (props) => {
   return (
     <div className="dictionaryDetails">
       <section className="backLink">
-        <Link
-          to="/home"
-          className="backLinkText"
-        >
-          <i className="fas fa-arrow-left"> <span className="span-text">Go Back Home</span></i>
+        <Link to="/home" className="backLinkText">
```

Comment:
Good job on this PR, but I think it is a best practice to name CSS classes without camelCase

Reply:
Something like this would be more appropriate back-link-text

Reply:
@Daramola98 Thanks for pointing this out, I think that we should create a ticket for this. Because this is not related to this ticket.